cli/program_state.rs

```diff
@@ -58,7 +59,13 @@ impl ProgramState {
     let dir = deno_dir::DenoDir::new(custom_root)?;
     let deps_cache_location = dir.root.join("deps");
     let http_cache = http_cache::HttpCache::new(&deps_cache_location);
-    let ca_file = flags.ca_file.clone().or_else(|| env::var("DENO_CERT").ok());
+    let ca_file_path =
+      flags.ca_file.clone().or_else(|| env::var("DENO_CERT").ok());
+
+    let ca_data: Option<String> = match ca_file_path.as_ref() {
+      None => None,
+      Some(ca_file_path) => Some(read_to_string(ca_file_path).unwrap()),
```

Comment:
This bit will panic if file doesn't exist, use `?` instead of `unwrap()`

Reply:
Cool thought of it but I was not that sure. I never worked with Rust before this PR but am definitely hooked